README.md

```diff
@@ -14,58 +14,55 @@ SPDX-License-Identifier: CERN-OHL-S-2.0
 
-## State of this repository
+If a single internet connection is available, it can be plugged into the nimble to provide free or paid internet access to anyone on the network. The offline services and content are always available to users, regardless of internet availability.
 
-We are excited to announce that we have received a grant from NLnet Foundation! With this generous funding our team is working with several incredible partners from the open hardware community to develop a smart documentation system which automatically generates all documentation and 3D models needed for any custom nimble configuration. 
+Anyone in the world can build their own nimble with locally sourced hardware and 3D-printed components.
 
-A sneek preview of the nimble builder:
 
-<p align="center"><img alt="6in NUC shelf" src="https://gitlab.com/gitbuilding/cadorchestrator/-/raw/master/assets/UIexample.png" height="auto" width="800"></p>
 
+<p align="center"><img alt="6in NUC shelf" src="https://raw.githubusercontent.com/Wakoma/nimble/refs/heads/master/assets/img/Model_M/36.jpg" height="auto" width="800"></p>
 
+## State of this repository
 
-This repository is in transition as we work towards the Smart-Doc goals of making all of the mechanical models parametric, and automatically generating the documentation for any configuration. To see repository before these changes, see the [post-hackathon release](https://github.com/Wakoma/nimble/releases/tag/Post-hackathon). This was the Nimble repository after the great work done at the [Open Toolchain Foundation](https://forum.opentoolchain.org/) hackathon.
+In 2024 we received a grant from NLnet Foundation to develop a smart documentation system which automatically generates all documentation and 3D models needed for any custom nimble configuration. 
 
-For more information on our project please see:
+<p align="center"><img alt="6in NUC shelf" src="https://nlnet.nl/logo/banner.png" height="auto" width="500"></p>
 
-* Jeremy's blog post: [First Nimble Project Update](https://7bindustries.com/blog/nimble_project_1.html)
-* This project is funded by  NLnet Foundation! See their announcemnt [here](https://nlnet.nl/project/HardwareManuals/).
+See the announcement [here](https://nlnet.nl/project/HardwareManuals/).
+Have a look at the other incredible projects they support [here](https://nlnet.nl/project/0).
 
 
-<p align="center"><img alt="6in NUC shelf" src="https://nlnet.nl/logo/banner.png" height="auto" width="500"></p>
 
+A sneek preview of the nimble builder:
 
-## Generating models locally
+<p align="center"><img alt="6in NUC shelf" src="https://gitlab.com/gitbuilding/cadorchestrator/-/raw/master/assets/UIexample.png" height="auto" width="800"></p>
 
-You can run the nimble code on your own computer. [Find out more about local model generation](generate.md)
+Access the CadOrchestrator repo [here!](https://gitlab.com/gitbuilding/cadorchestrator).
```

Comment:
We may want to be a bit clearer what CadOrchestrator is.

```suggestion
To create our interactive documentation generator we ended up creating an entirely new library called [CadOrchestrator](https://gitlab.com/gitbuilding/cadorchestrator).
```

Reply:
@ericnitschke If you are happy with this you can click the commit suggestion and it will commit it into your branch automatically.